bindings/java/src/org/sleuthkit/datamodel/blackboardutils/FileAttachment.java

```diff
@@ -74,10 +74,15 @@ public FileAttachment(SleuthkitCase caseDb, Content dataSource, String pathName)
 
 		//normalize the slashes.
 		this.filePathName = normalizePath(pathName);
-
+		
+		
 		String fileName = filePathName.substring(filePathName.lastIndexOf('/') + 1);
-		String parentPathSubString = filePathName.substring(0, filePathName.lastIndexOf('/'));
+		if (fileName.isEmpty()) {
+			throw new TskCoreException(String.format("No name specified for attachment: %s", pathName ));
+		}
```

Comment:
Please add the data source object ID to the exception message as well as the file path.